docs/optimization_guide/memory_optimization_guide.md

```diff
@@ -0,0 +1,23 @@
+# Optimizing memory usage {#openvino_docs_memory_optimization_guide}
+
+Before applying any of provided recomendations, please, note that it may significantly impact first inference latency. 
+
+The most RAM-consuming OpenVINO stage is model compilation. It may cause several issues:
+
+* Not enough memory to compile model. To decrease memory requirement next options may be applied: 
+  
+  * IR mapping - we introduced memory mapping (using `mmap`) as a default way to operate with IR. Because of "memory on demand" nature, there is no need to store weights fully in RAM, so it decreases memory level required for compilation. Also `mmap` provides extensive memory sharing, so next compilation of same model will fetch same memory from RAM instead of additional read from storage
```

Comment:
I propose to rephrase this. That we introduced `mmap` support IR frontend already supports it

Reply:
I think we also can notice the option `enable_mmap`

Reply:
Done, please check